dev/com.ibm.websphere.appserver.features/visibility/public/concurrent-1.0/com.ibm.websphere.appserver.concurrent-1.0.feature

```diff
@@ -23,3 +23,5 @@ Subsystem-Name: Concurrency Utilities for Java EE 1.0
 kind=ga
 edition=core
 WLP-Activation-Type: parallel
+WLP-InstantOn-Enabled: true
```

Comment:
I think I am ok adding this here as long as we test concurrent-1.0 with checkpoint.  I think we do test it for EE 10 because it is included in EE 10 webprofile.